cpg_pipes/hb/batch.py

```diff
@@ -0,0 +1,81 @@
+"""
+Extending the Hail's `Batch` class.
```

Comment:
That sounds like you're deriving a class, but this is more like a set of utility functions for creating new batches, right?

Reply:
I dropped the class from this initial PR for simplicity.

In the `initial` branch, I derived the `Batch` class so I can override `b.new_job()` and count jobs and job types before submission, as a good sanity check for pipelines: https://github.com/populationgenomics/production-pipelines/blob/initial/cpg_pipes/hb/batch.py
It's not a critical feature, and in this PR I wanted to keep only the basic things that would make the VEP example running.